app/views/coronavirus_landing_page/components/landing_page/_header-notice.html.erb

```diff
@@ -0,0 +1,32 @@
+<section class="covid-header-notice" aria-label="notice" role="region">
+  <div class="covid-header-notice__header">
+    <%= render("coronavirus_landing_page/components/shared/nhs_logo") %>
+  </div>
+
+  <div class="covid-header-notice__content">
+    <% if heading.present? %>
+      <h2 class="covid-header-notice__heading govuk-heading-m"><%= heading %></h2>
+    <% end %>
+    <% if list.any? %>
+      <ul class="covid-header-notice__list govuk-list govuk-list--bullet">
+        <% list.each do |item| %>
+          <li><%= item %></li>
+        <% end %>
+      </ul>
+    <% end %>
+
+    <% if call_to_action.present? %>
+      <%= link_to(call_to_action[:title], call_to_action[:href], class: "covid-header-notice__call-to-action govuk-link") %>
+    <% end %>
+  </div>
+
+  <div class="covid-header-notice__footer">
+    <%= render "govuk_publishing_components/components/action_link", {
+      text: "Placeholder text",
+      subtext: "More placeholder text",
+      subtext_href: "/page-not-provided-yet",
+      mobile_subtext: true,
+      nhs_icon: true
+    } %>
```

Comment:
What is the purpose of this placeholder? Is it worth adding a comment explaining it's purpose and presumably when it will be replaced. For example:
```
<%# TODO: Replace this placeholder when ..... %>
```